From e9710debf12bde199abe7ba548b0dc080bc346a0 Mon Sep 17 00:00:00 2001
From: Munehisa Kamata <kamatam@amazon.com>
Date: Fri, 17 Mar 2017 10:15:25 +0000
Subject: drivers/amazon: add network device drivers support

This is a place holder for network device driver updates.

Signed-off-by: Munehisa Kamata <kamatam@amazon.com>
Reviewed-by: Cristian Gafton <gafton@amazon.com>
Reviewed-by: Guru Anbalagane <guruanb@amazon.com>

CR: https://cr.amazon.com/r/6834581/
---
 drivers/amazon/Makefile     | 1 +
 drivers/amazon/net/Makefile | 3 +++
 2 files changed, 4 insertions(+)
 create mode 100644 drivers/amazon/net/Makefile

diff --git a/drivers/amazon/Makefile b/drivers/amazon/Makefile
index 6b4996d..fc5f70d 100644
--- a/drivers/amazon/Makefile
+++ b/drivers/amazon/Makefile
@@ -1,3 +1,4 @@
 #
 # Amazon Driver Updates
 #
+obj-$(CONFIG_AMAZON_DRIVER_UPDATES)	+= net/
diff --git a/drivers/amazon/net/Makefile b/drivers/amazon/net/Makefile
new file mode 100644
index 0000000..6b4996d
--- /dev/null
+++ b/drivers/amazon/net/Makefile
@@ -0,0 +1,3 @@
+#
+# Amazon Driver Updates
+#
-- 
2.7.5

